src/Parser/TemplateParser.cs

```diff
@@ -14,17 +14,17 @@ public static PartList Parse(string template)
             return parser.Parse();
         }
 
-        private static readonly char[] HoleDelimiters = { ',', ':', '}' };
-        private static readonly char[] AlignmentDelimiters = { ':', '}' };
+        private static readonly char[] HoleDelimiters = { ':', '}' ,',' };
         private static readonly char[] TextDelimiters = { '{', '}' };
         private readonly string _template;
         private readonly int _length;
         private int _position;
         private int _holeIndex;
-        private PartList _parts;
+        private readonly PartList _parts;
 
         private TemplateParser(string template)
         {
+            _parts = new PartList();
```

Comment:
Yes, I agree. It was remanent of an old code structure I had before. Given that the ctor is private and called from inside the `Parse()` method, there's no chance you allocate an unused object ;)

I was saying that I would even move it to 
`private PartList _parts = new PartList();`